feat: Added HTML output option (#76)
DenverCoder1/github-readme-youtube-cards: README.md

| `commit_message`      | The commit message to use for the commit          | "docs(readme): Update YouTube cards"                    |
| `readme_path`         | The path to the README file                       | "README.md"                                             |
| `output_only`         | Whether to skip writing to the readme file        | "false"                                                 |

> | `output_type`         | The output syntax to be used by the action        | "markdown"                                              |

Reviewer: ```suggestion
| `output_type`         | The output syntax to use ("markdown" or "html")   | "markdown"                                              |
```
